src/examples/text-inputs/Inline.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Field, Label, Input } from '@zendeskgarden/react-forms';
+import styled from 'styled-components';
+
+const StyledField = styled(Field)`
+  display: flex;
+  align-items: center;
+`;
+
+const StyledLabel = styled(Label)`
+  /* stylelint-disable-next-line property-no-unknown */
```

Comment:
Please knock out the `property-no-unknown` rule in `website`'s stylelint config and remove the `stylelint-disable` comment here. Note that eslint/stylelint disable comments are distracting in example codez.